### ISSUE TITLE
latest values.yaml mentions 1.13.0-alpine but that doesn't exist in dockerhub

### ISSUE THREAD

**Issue by zebehringer** [vernemq/docker-vernemq#355]
having upgraded to the latest Helm chart results in an image pull failure

**Comment by ioolkos** [comment]
@zebehringer just pushed `1.13.0-alpine` to Dockerhub. Experimenting with multiarchitecture builds, let me know when something does not work as expected.

---
:point_right: Thank you for supporting VerneMQ: https://github.com/sponsors/vernemq
:point_right: Using the binary VerneMQ packages commercially (.deb/.rpm/Docker) requires a paid subscription.

**Comment by ioolkos** [comment]
@zebehringer Yes, I'll check whether we can build for `darwing/arm64` too. Currently only `linux/arm64` is uploaded.

---
:point_right: Thank you for supporting VerneMQ: https://github.com/sponsors/vernemq
:point_right: Using the binary VerneMQ packages commercially (.deb/.rpm/Docker) requires a paid subscription.

**Comment by zebehringer** [comment]
I was able to get the latest image running on my M1 explicitly using the `linux/arm64` platform, so I'm happy with that. Thanks again!

**Comment by ioolkos** [comment]
@zebehringer great to hear, thanks for your feedback!

---
:point_right: Thank you for supporting VerneMQ: https://github.com/sponsors/vernemq
:point_right: Using the binary VerneMQ packages commercially (.deb/.rpm/Docker) requires a paid subscription.
